modules/aggs-matrix-stats/src/main/java/org/elasticsearch/search/aggregations/matrix/stats/InternalMatrixStats.java

```diff
@@ -7,6 +7,8 @@
  */
 package org.elasticsearch.search.aggregations.matrix.stats;
 
+import joptsimple.internal.Strings;
```

Comment:
I don't think we mean for jopt to be used outside of the cli bits.

Reply:
🤦